test/types/index.ts

```diff
@@ -1,207 +1,221 @@
-
 import * as env from '../../';
-import { expect } from 'chai'
-import 'mocha'
-import { assert, IsExact } from 'conditional-type-checks'
+import { expect } from 'chai';
```

Comment:
Can you revert the formatting changes to this file? Thanks!

Reply:
whoops